src/reader.js

```diff
@@ -313,7 +313,9 @@ Reader.prototype.bytes = function read_bytes() {
     if (Array.isArray(this.buf)) // plain array
         return this.buf.slice(start, end);
     return start === end // fix for IE 10/Win8 and others' subarray returning array of size 1
-        ? new this.buf.constructor(0)
+        ? (typeof Buffer.alloc === "function")
```

Comment:
Will this work if there's no `Buffer` at all (in non-Node environments)? I'm afraid it might fail with `ReferenceError`.

Maybe try something like `typeof Buffer !== "undefined" && this.buf.constructor === Buffer && typeof Buffer.alloc === "function"`?

Reply:
Ok, new commit apended.